fortanix-vme/fortanix-vme-eif/build.rs

```diff
@@ -0,0 +1,34 @@
+#![deny(warnings)]
+use blobs_tool::download::download_blobs;
+use blobs_tool::utils::create_symlink;
+use blobs_tool::BLOB_DEFAULT_DIR;
+use std::env;
+use std::path::{Path, PathBuf};
+
+fn main() {
+    println!("cargo:rerun-if-env-changes=CARGO_MANIFEST_DIR");
+
+    let mut test_dir = PathBuf::from(format!(
+        "{}/tests/data",
+        env::var("CARGO_MANIFEST_DIR").expect("cargo manifest dir"),
+    ));
+    let output_dir = Path::new(BLOB_DEFAULT_DIR);
+    for blob in download_blobs(output_dir).unwrap() {
+        println!("cargo:rerun-if-changed={}", blob);
+        create_symlink(&blob, &mut test_dir).unwrap();
```

Comment:
This seems unnecessarily complex. Can't we move this logic to download the aws blobs to `blobs-tools` (maybe rename it to `aws-nitro-blobs`), expose the blobs using a ` pub const KERNEL: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/bzImage"));` and create a dependency on that crate? Then only one build script on the `aws-nitro-blobs` crate is required

Reply:
Added `aws-nitro-blobs` crate and kept `blobs-tools` (renamed it `aws-nitro-blobs-downloader`). I needed a separate crate for downloading as downloaded files are consumed in `aws-nitro-blobs/lib.rs`. Alternatively, i can move all functions from `aws-nitro-blobs-downloader` to `aws-nitro-blobs`'s `build.rs` but preferred to keep it as separate crate.